### PR TITLE
add padding top on featured image fix #100

### LINKED ISSUE #100: featured image gets pushed down by text
this is around 688x1115

### PR DESCRIPTION
**Pull request by mixcas**
This fixes #100 and #94 

I fucked up i should have done diff PRs, sorry.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -324,13 +324,16 @@ Featured = {
...
+      // Get all classes except for s* ones
+      var cleanClasses = $titleHolder.attr('class').replace(/s\d+/,'');
+
+      // Replace title classes with clean ones
+      $titleHolder.removeClass().addClass(cleanClasses);
 
       // Set col size
-      for (var i = titleCol; $titleHolder.width() < $('#featured-post-title').width() && i < 9; i++ ) {
+      var titleCol = 0;
+      for (var i = 0; $titleHolder.width() < $('#featured-post-title').width() && i < 6; i++ ) {
```
Comment (varffs) [inline js/main.js]: does this mean the first iteration of this code is for `.col-0`?

File: js/main.js [hunk 2]
```diff
@@ -342,13 +345,14 @@ Featured = {
...
 
       if( $subtitleHolder.length ) {
-        var subtitleCol = $subtitleHolder.attr('class').replace('col s','');
+        // Get all classes except for s* ones
+        var cleanClasses = $subtitleHolder.attr('class').replace(/s\d+/,'');
 
-        // Turn string into int
-        subtitleCol *= 1;
+        // Replace title classes with clean ones
+        $subtitleHolder.removeClass().addClass(cleanClasses);
```
Comment (varffs) [inline js/main.js:352]: is some of this code duplicated and worth making a function?

File: js/main.js [hunk 3]
```diff
@@ -324,13 +324,16 @@ Featured = {
...
       var $titleHolder = $('#featured-post-title-holder');
-      var titleCol = $titleHolder.attr('class').replace('col s','');
 
-      // Turn string into int
-      titleCol *= 1;
+      // Get all classes except for s* ones
+      var cleanClasses = $titleHolder.attr('class').replace(/s\d+/,'');
+
+      // Replace title classes with clean ones
+      $titleHolder.removeClass().addClass(cleanClasses);
```
Comment (varffs) [inline js/main.js:332]: is this the bit that allows the scaling down then up again to work?

Comment (mixcas) [inline js/main.js:332]: yeah, this part removes `.s*`
then `for`s are starting `for` from 1

**Pull request merged**